examples/with-doge/src/createDogeTx.ts

```diff
@@ -0,0 +1,254 @@
+);
+const N2 = N >> 1n;
+
+function derToRS(derHex: string): { r: string; s: string } {
+  const b = Buffer.from(strip0x(derHex), "hex");
+  if (b.length < 2 || b[0] !== 0x30) throw new Error("Bad DER: no sequence");
+
+  let off = 1;
+  let len: number = b[off++]!;
+  if (len & 0x80) {
+    const n = len & 0x7f;
+    len = 0;
+    for (let i = 0; i < n; i++) {
+      if (off >= b.length) throw new Error("Bad DER: length overflow");
+      len = (len << 8) | b[off++]!;
+    }
+  }
+
+  if (b[off++] !== 0x02) throw new Error("Bad DER: r tag");
+  const rlen: number = b[off++]!;
+  if (off + rlen > b.length) throw new Error("Bad DER: r value out of range");
+  const r = b.subarray(off, off + rlen);
+  off += rlen;
+
+  if (b[off++] !== 0x02) throw new Error("Bad DER: s tag");
+  const slen: number = b[off++]!;
+  if (off + slen > b.length) throw new Error("Bad DER: s value out of range");
+  const s = b.subarray(off, off + slen);
+
+  return { r: r.toString("hex"), s: s.toString("hex") };
+}
+
+function toLowS(rHex: string, sHex: string) {
+  const r = BigInt("0x" + strip0x(rHex));
+  let s = BigInt("0x" + strip0x(sHex));
+  if (s > N2) s = N - s;
+  return {
+    r: r.toString(16).padStart(64, "0"),
+    s: s.toString(16).padStart(64, "0"),
+  };
+}
+
+function rsToDer(rHex: string, sHex: string): string {
+  function trim(buf: Buffer): Buffer {
+    let i = 0;
+    while (i < buf.length - 1 && buf[i] === 0) i++;
+    const v = buf.subarray(i);
+    if (v.length === 0) throw new Error("Bad DER: empty integer");
+    return v[0]! & 0x80 ? Buffer.concat([Buffer.from([0x00]), v]) : v;
+  }
+  const r = trim(Buffer.from(strip0x(rHex), "hex"));
+  const s = trim(Buffer.from(strip0x(sHex), "hex"));
+  const rSeq = Buffer.concat([Buffer.from([0x02, r.length]), r]);
+  const sSeq = Buffer.concat([Buffer.from([0x02, s.length]), s]);
+  const body = Buffer.concat([rSeq, sSeq]);
+  return Buffer.concat([Buffer.from([0x30, body.length]), body]).toString(
+    "hex",
+  );
+}
+
+// Turnkey signing, NO_OP over 32-byte digest
+async function turnkeySignDigestHex(digestHex: string): Promise<string> {
+  const tk = new TurnkeyServerSDK({
+    apiBaseUrl: process.env.BASE_URL!,
+    apiPrivateKey: process.env.API_PRIVATE_KEY!,
+    apiPublicKey: process.env.API_PUBLIC_KEY!,
+    defaultOrganizationId: process.env.ORGANIZATION_ID!,
+  });
+
+  const resp: any = await tk.apiClient().signRawPayloads({
+    signWith: SIGN_WITH!,
+    payloads: [digestHex],
+    encoding: "PAYLOAD_ENCODING_HEXADECIMAL",
+    hashFunction: "HASH_FUNCTION_NO_OP",
+  });
+
+  const sig0 = resp?.signatures?.[0] || resp?.result?.signatures?.[0];
+  if (!sig0) throw new Error("Turnkey: no signatures");
+
+  if (sig0.r && sig0.s) {
+    const { r, s } = toLowS(sig0.r, sig0.s);
+    return rsToDer(r, s);
+  }
+  if (typeof sig0.signature === "string") {
+    const { r, s } = derToRS(sig0.signature);
+    const low = toLowS(r, s);
+    return rsToDer(low.r, low.s);
+  }
```

Comment:
I don't think this will ever happen? Turnkey API always returns r and s values. So this should be good to remove, and you can also remove the `derToRS` helper?